src/utils/signMessage.js

```diff
@@ -0,0 +1,39 @@
+import { ethers } from "ethers";
+import { Keypair } from "@solana/web3.js";
+
+export async function signMessage(message, privateKey, chainType = "evm") {
```

Comment:
We don't need this, it should reference the `src/services/wallet/index.ts` just like every other function